internal/nvsandboxutils/gpu-resources.go

```diff
@@ -0,0 +1,74 @@
+/**
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+**/
+
+package nvsandboxutils
+
+import (
+	"strings"
+)
+
+import "C"
+
+type GpuResource struct {
+	Version uint32
+}
+
+//	type GpuFileInfo_v1 struct {
+//		Next        *GpuFileInfo_v1
+//		FileType    uint32
+//		FileSubType uint32
+//		Module      uint32
+//		Flags       uint32
+//		FilePath    *int8
+//	}
```

Comment:
Is this commented block required?

Reply:
No, I think this can be removed now. Thanks!